### ISSUE TITLE
osvr_display_config_built_in_osvr_hdk20_v1 distortion mesh doesn't work well enough

### ISSUE THREAD

**Issue by lonetech** [sensics/OSVR-RenderManager#181]
While it corrects some gross geometric distortions toward the corners, it produces a drastic shift in angles with a circular shape, and some minor CA also. The circular distortion change has been described as a bubble or fisheye effect. I tried simply disabling the mesh because I wasn't sure it even worked!

**Comment by russell-taylor** [comment]
Agreed.  We've got two engineers working on that now, doing a camera-based calibration instead of a theoretical-parameters-based calibration.

**Comment by lonetech** [comment]
I was thinking about trying to map the distortion. I figured a per channel sequence of binary refinement combined with a synchronized high resolution camera, for instance a networked mobile phone, could be used to build a displacement map given a few minutes. The tricky portions might be placing it and doing reasonable averaging for a range of placements (I expect, for instance, I'd never get it head on, and I'd first calculate rectification of the camera using Hugin). But I guess you've already got a better start.

**Comment by russell-taylor** [comment]
Our current work is at https://github.com/sensics/Optical-measurements and it is based on work at https://github.com/OSVR/distortionizer/tree/master/angles_to_config whose operation is described at https://github.com/OSVR/distortionizer/blob/master/angles_to_config/doc/anglesToConfig.md with the theory described at a link pointed to by that last document.  We're doing basically as you suggest, and are close to a solution.  We're not using a mobile phone app; if you build one let us know because that could be really cool.
